xcrypto/hash_test.go

```diff
@@ -161,6 +162,9 @@ func TestHash_BinaryAppender(t *testing.T) {
 			// Append binary data to the prebuilt slice
 			state, err := hashWithBinaryAppender.AppendBinary(prebuiltSlice)
 			if err != nil {
+				if strings.Contains(err.Error(), "hash state is not marshallable") {
```

Comment:
Remove the `t.Skip("not supported")` code from above.

Reply:
updated PTAL